maestro-client/src/main/java/maestro/drivers/IOSDriver.kt

```diff
@@ -494,6 +495,10 @@ class IOSDriver(
         iosDevice.setPermissions(appId, permissions)
     }
 
+    override fun fetchSessionId(): UUID {
+        return iosDevice.fetchSessionInfo().expect {  }
```

Comment:
This will fail if fetchSessionInfo errors. Is this intended?

Reply:
Yes